cli/tests/program.rs

```diff
@@ -860,7 +860,10 @@ fn test_cli_program_upgrade_auto_extend() {
          RPC response error -32002: \
          Transaction simulation failed: \
          Error processing Instruction 0: \
-         account data too small for instruction [3 log messages]",
+         account data too small for instruction 3 log messages:\n  \
```

Comment:
Last nit: looking at this, it feels like we need some kind of punctuation between the error display and the `## log messages`. (It would be easy to misread this is `instruction 3`.)
Could do this in RpcResponseErrorData Display, but might be better [on RpcError itself](https://github.com/anza-xyz/agave/blob/d8e9da5b5d02f6ee951e597dbb8e6c6fd5e32ff7/rpc-client-api/src/request.rs#L259-L260), something like:
```
#[error("RPC response error {code}: {message}, {data}")]
```
Or be more forceful with a semicolon or dash

Reply:
That's a good point. Forceful is probably best to make this easier to parse, so it's clear that it's not part of the error message. I'm leaning towards semicolon